span.go

```diff
@@ -77,6 +77,14 @@ ReferencesLoop:
 	sp.raw.Start = startTime
 	sp.raw.Duration = -1
 	sp.raw.Tags = opts.Options.Tags
+
+	if tracer.opts.MetaEventLogging && IsMetaSpan(sp) {
```

Comment:
Should we include the starting tags in the meta event? I'm imagining that if we were filtering a view and wanted to find unfinished spans, it would be helpful to know the tags of the unfinished span.

Reply:
I think I'd like to hold off on increasing the size of these meta spans for now. I think we should keep it mind for the future though.